pmda/rmsf.py

```diff
@@ -159,8 +159,10 @@ def _prepare(self):
         self.mean = self.sumsquares.copy()
```

Comment:
Do you still need `self.sumsquares` and `self.mean` in `_prepare`??

Can you get rid of `_prepare` completely?

Reply:
No, I don't think it's necessary to keep it. I ran it with `_prepare` commented out, and it still passes all of the tests, so I don't think it's necessary.